cdap-docs/examples-manual/source/examples/wikipedia-data-pipeline.rst

```diff
@@ -86,6 +86,9 @@ This application demonstrates:
   and the other using K-Means. The application is packaged with the two possible application config JSON files at
   *resources/wikipedia-kmeans.json* and *resources/wikipedia-lda.json* in the application directory.
 
+- **Cross-namespace data access:** runtime arguments are used to pass in a namespace allowing each run to be customized
```

Comment:
Few corrections: 
1. Programs take namespace as runtime arguments/preferences not applications so it will be better to explicitly say that and say they are doing cross namespace. 
2. Include the key for the runtime arguments.
3. We should also say what it means for this application's behavior. Currently what we have is very generic and seems like a definition for cross namespace.